web/modules/custom/joinup_search/joinup_search.behat.inc

```diff
@@ -331,6 +346,37 @@ class SearchSubContext extends DrupalSubContextBase implements DrupalSubContextI
     $this->getSession()->visit($this->locatePath('/search?keys=' . $keywords));
   }
 
+  /**
+   * Press a key in the search field.
+   *
+   * Works only in Javascript-enabled browsers.
+   *
+   * @param string $key
+   *   The human readable name of the key to press.
+   *
+   * @throws \Exception
+   *   Thrown when the browser doesn't support Javascript or when the search
+   *   field is not found.
+   *
+   * @When I press :key in the search field
+   */
+  public function pressKeyInSearchField(string $key): void {
+    if (!$this->browserSupportsJavascript()) {
```

Comment:
We have `\Drupal\joinup\Traits\BrowserCapabilityDetectionTrait::assertJavaScriptEnabledBrowser` just for this :)

Reply:
Good suggestion! Fixed.